v2/cdc-parent/cdc-embedded-connector/src/main/java/com/google/cloud/dataflow/cdc/connector/App.java

```diff
@@ -104,9 +103,6 @@ public static DebeziumToPubSubDataSender setup(String[] args) throws Configurati
 
     // Printing the information about the bindings for SLF4J:
     Configuration config = getConnectorConfiguration(args);
-    final StaticLoggerBinder binder = StaticLoggerBinder.getSingleton();
-    System.out.println("Logger Binding: " + binder.getLoggerFactory());
-    System.out.println(binder.getLoggerFactoryClassStr());
```

Comment:
Similarly, why did we need this - was this causing errors?

Reply:
Sounds good, thanks! That makes sense and should be fine for validation at least.